### PR TITLE
Remove unused files.

### PR DESCRIPTION
**Pull request by Empowerful**
## Description (optional)

Removes unused files and adds a check to Webpack the warn about any unused files in the future.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
     "ci:test:core:components": "yarn wsrun --serial --exclude-missing ci:test",
     "ci:test:frontend": "yarn workspace blockchain-wallet-v4-frontend ci:test:frontend",
     "clean": "cross-env yarn wsrun clean && rimraf build && rimraf coverage && rimraf dist && rimraf *.log && rimraf node_modules",
+    "commit": "git cz --gpg-sign",
```
Comment (schnogz) [inline package.json]: custom commit cmd?  please remove

Comment (schnogz) [inline package.json]: custom commit cmd? please remove

**Comment by schnogz** [comment]
@plondon Do you know what the whitelist files are? example => `assets/locales/whitelists/*`

**Comment by plondon** [comment]
Whitelist files are autogenerated by localization tool. Fine to remove.

File: package.json (hunk 1 above)
Comment (Empowerful) [reply package.json]: I added it at [the suggestion of the Commitizen documentation](https://github.com/commitizen/cz-cli#if-your-repo-is-commitizen-friendly).

Do you not think it's useful for making it easier for developers to comply with Commitizen?

**Pull request merged**